src/test/scala/com/softwaremill/realworld/users/UsersRepositorySpec.scala

```diff
@@ -87,6 +87,64 @@ object UsersRepositorySpec extends ZIOSpecDefault:
           v <- repo.add(UserRegisterData(email = "test@test.com", username = "tested", password = "tested"))
         } yield zio.test.assert(v)(isUnit) // TODO check DB?
       }
+    ),
+    suite("update user")(
+      test("update user bio") {
+        for {
+          repo <- ZIO.service[UsersRepository]
+          _ <- repo.add(UserRegisterData(email = "test@test.com", username = "tested", password = "tested"))
+          user <- repo.updateByEmail(UserUpdateData(None, None, None, Some("Updated test bio"), None), "test@test.com")
+        } yield zio.test.assert(user) {
+          Assertion.isSome {
+            Assertion.hasField[UserData, Option[String]](
```

Comment:
**praise:** Nice, I couldn't figure out how to use this, the documentation is a bit messy and out of date :)